.github/workflows/ci.yml

```
@@ -23,35 +46,32 @@ jobs:
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install -r project/requirements.txt
+          pip install -r requirements-dev.txt
+
+      - name: Full lint check
+        run: |
+          flake8 project/ --count --exit-zero --max-complexity=10 --max-line-length=88 --statistics
+
+      - name: Fast test (critical tests only)
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2155:warning:1:8: Declare and assign separately to avoid masking return values

(shellcheck)